configure.ac

```diff
@@ -339,7 +341,9 @@ AC_ARG_ENABLE([jack],
 if [ test x$jack_framework != xyes ] && [ test x$jack = xyes ] ; then
     AC_CHECK_LIB([rt], [shm_open], [LIBS="$LIBS -lrt"])
     AC_CHECK_LIB([jack], [jack_set_xrun_callback], [JACK_LIBS="-ljack" ; jack=xrun])
-    AC_CHECK_LIB([jack], [jack_set_error_function], [JACK_LIBS="-ljack" ; jack=yes], [jack=no])
+    AC_CHECK_LIB([jack], [jack_set_error_function], [JACK_LIBS="-ljack" ; jack=yes],
+        [AC_MSG_WARN([JACK development files not found... skipping (See INSTALL.txt)])
+        jack=no])
 fi
```

Comment:
i'm afraid, that people (at least those, who don't read `INSTALL` files...)  will not see this information.
probably the warning should be emitted at the very end of the configure-script (*after* the summary at the end). with something like:

~~~
 AS_IF([test x$enable_jack = xyes -a x$jack != xyes],
    AC_MSG_WARN([JACK requested but not found. (See INSTALL.txt)])
)
~~~

Reply:
Well, the last example from the email thread included copy/paste from the configure output, so it comes *some* people would see it.

Reply:
cool. so shall we remove the summary at the end of `configure` (because those who read it, can also just scan through the output of configure)?

Reply:
> remove the summary at the end of `configure`

please don't ;-)
moving the warning(s) to the end of the summary seems sensible to me.